v8_c_api/src/v8_c_api.cpp

```diff
@@ -605,6 +605,12 @@ v8_local_string* v8_NewString(v8_isolate* i, const char *str, size_t len) {
     return v8_str;
 }
 
+v8_local_string* v8_CloneString(v8_local_string *source) {
+    v8_local_string *v8_str = (struct v8_local_string*)V8_ALLOC(sizeof(*v8_str));
+    v8_str = new (v8_str) v8_local_string(*source);
```

Comment:
Will this only clones the local reference? Or will it also clones the underline JS string object?

Reply:
Yes I believe its OK but we should probably mention it on the docs?